Utilities/DataLoading/EdFi.LoadTools/ApiClient/XsdMetadata/XsdFilesRetriever.cs

```diff
@@ -42,22 +43,22 @@ public class XsdFilesRetriever : IXsdFilesRetriever, IDisposable
             _odsVersionInformation = odsVersionInformation;
             _httpClient = new HttpClient {Timeout = new TimeSpan(0, 0, 5, 0)};
 
-            _xsdMedataUrl = _configuration.GetValue<string>("OdsApi:XsdMetadataUrl");
-            _extension = _configuration.GetValue<string>("OdsApi:Extension");
+            _xsdMedataUrl = _configuration.XsdMetadataUrl;
+            _extension = _configuration.Extension;
 
             SetXsdFolder();
 
             void SetXsdFolder()
             {
                 string currentDirectory = Directory.GetCurrentDirectory();
 
-                string workingFolder = string.IsNullOrEmpty(_configuration.GetValue<string>("Folders:Working"))
+                string workingFolder = string.IsNullOrEmpty(_configuration.WorkingFolder)
                     ? currentDirectory
-                    : Path.GetFullPath(_configuration.GetValue<string>("Folders:Working"));
+                    : Path.GetFullPath(_configuration.WorkingFolder);
 
-                _xsdFolder = string.IsNullOrEmpty(_configuration.GetValue<string>("Folders:Xsd"))
+                _xsdFolder = string.IsNullOrEmpty(_configuration.XsdFolder)
                     ? Path.Combine(workingFolder, "xsd")
-                    : Path.GetFullPath(_configuration.GetValue<string>("Folders:Xsd"));
+                    : Path.GetFullPath(_configuration.XsdFolder);
```

Comment:
I figured with the change to use the BulkLoadClientConfiguration these ternaries could go away and you could just use the properties from the config directly.

Reply:
right, i have them in the configuration.